CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release 0.1.0
+
++ [#2] Added extra functionality to jest specific to MB3
```

Comment:
Are you trying to link to https://github.com/mb3online/jest-speck-plugin/issues/2?

```
[#2](https://github.com/mb3online/jest-speck-plugin/issues/2)
```